src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -168,8 +169,8 @@ float64_t CCrossValidation::evaluate_one_run(
 			/* evtl. update xvalidation output class */
 			CrossValidationFoldStorage* fold = new CrossValidationFoldStorage();
 			SG_REF(fold)
-			fold->set_run_index(index);
-			fold->set_fold_index(i);
+			fold->put("run_index", (index_t)index);
+			fold->put("fold_index", i);
```

Comment:
just "fold" (imo, you can object of course ;) )

Reply:
I would prefer to keep the `_index` part because it gives more information about what the user will get. I mean, if I do like `get("fold")`, what will I get? A complex object, an `int64_t`? with index is more "obvious" imho

Reply:
The way I see this is: Labels should be short to reduce typing. The user once needs to read the description (thats why we have it!) and then wants to type less. Better than alway forcing people to type imo. And also less redundant. It is a taste thing though I appreciate that. Fine if we disagree, your call and we can also change it later. I just give my opinion :)

Reply:
I would prefer to change the naming at the end then :)

Reply:
list is growing :) but yeah good point!